src/index.html

```diff
@@ -8,5 +8,43 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <div class="form">
```

Comment:
You dont need this extra block for your calendar. otherwise you should make your calendar an element of the form block)